src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -551,6 +564,10 @@ QJsonArray mp::LXDVMImageVault::retrieve_image_list()
     {
         // ignore exception
     }
+    catch (const std::exception& e)
+    {
+        mpl::log(mpl::Level::warning, category, e.what());
+    }
```

Comment:
Catch-all on purpose? With the richer exceptions now, couldn't we get rid of the catch-all further up, too?

Reply:
I suppose this could be a `LocalSocketConnectionException` and let other exception types bubble up.  I'm unsure what catch-all further up you are referring to.

Reply:
Changed to `LocalSocketConnectionException`.